CMakeLists.txt

```diff
@@ -7,8 +7,8 @@ if(NOT ${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
 endif()
 set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra")
 
-# Use C++11
-set(CMAKE_CXX_STANDARD 11)
+# Use C++14
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
In the workspace, other packages set the compiler to c++11, for example:
https://github.com/machines-in-motion/shared_memory/blob/master/CMakeLists.txt#L8
I guess the clean things to do would be to update all those as well. I'd propose to change the c++ version in all package and directly push to master, after creating a c++11 tag. And check right away what happens in bamboo.

Reply:
:+1: